Source/OCMockTests/OCMockObjectPartialMocksTests.m

```diff
@@ -608,5 +608,14 @@ - (void)testMethodSwizzlingWorksForVoidReturns
 	XCTAssertNoThrow([foo method1], @"Should have worked.");
 }
 
+- (void)testAttemptingToVerifyMethodImplementedByNSObjectUsingMacroThrows
+{
+	TestClassThatCallsSelf *realObject = [[TestClassThatCallsSelf alloc] init];
+	id mock = [OCMockObject partialMockForObject:realObject];
```

Comment:
If you add `[[mock stub] categoryMethod];` here, then the verification will work and the exception is not thrown.

Reply:
Yes, which is an easy fix, but not clear at all that it is required IMHO.